docs/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+    Copyright 2018-2017 EntIT Software LLC, a Micro Focus company.
```

Comment:
Change the project inception year in `caf-common-parent` back to 2015.  It should be the year that the project started rather than the year that the module itself was created.  (Or you could override the inception year in the aggregator to set it to 2015).